### ISSUE TITLE
Hair color tooltip is random

### ISSUE THREAD

**Issue by thelabcat** [animate1978/MB-Lab#339]
**Describe the bug**
When hovering over items in the hair color menu, the tooltip displayed is entirely random, and is not even hair color most of the time (IE: "plum.")

**To Reproduce**
1. Create and finalize a character.
2. Select the mesh part of the character, and go to the MB-Lab hair panel.
3. Select the hair color pull down, and hover over an option. The tooltip displayed does not match the entry, and is sometimes different each time for the _same_ entry.

**Expected behavior**
Hair color tooltips that possibly describe the color involved, such as "Strawberry blonde: Blonde, but close to the 'red hair' color."

**Additional context**
I may be using a slightly out of date version of the addon, but I don't think I am.

**Comment by animate1978** [comment]
Just a question, which render engine are you using by chance? Depending on the engine (Cycles / EEVEE) will result in different shaders loaded into the scene, not to mention if you switch render engines at any point after creation of the character.

**Comment by thelabcat** [comment]
Probably EEVEE, but I think it happens with both. Will check...

**Comment by thelabcat** [comment]
OK, I didn't even change anything and just open the default blend file, and hovering over "jet black," ￼it said "Auburn," then I moved a little and it said "_reconstruct," something else, something else still (definitely not colors), and finally settled on a blank tooltip. I will say that for awhile nothing in the list was showing any tooltips, so maybe they got backlogged? I did hover over auburn before jet black.

**Comment by thelabcat** [comment]
Yeah, same thing again. Long abscesses of any tooltips, then when they come, it's a random color, "_reconstruct." or some other programming sounding phrase.

Confirmed, different tooltips show for the same item when I hover back over it without even closing the list.

**Comment by thelabcat** [comment]
More items:
Some non-ASCII goblygook.
"arr_1."
"MELANIN."
"arr_0."
No tooltip at all.

Burgundy seems to display "Auburn" more often than it displays anything else. Are you using sets somewhere? They have arbitrary order and could cause something like this.

Aha, the console has been very busy.
![image](https://user-images.githubusercontent.com/53358500/111057272-39656e00-8454-11eb-932c-d10b5f3e991e.jpeg)
![image](https://user-images.githubusercontent.com/53358500/111057273-3ec2b880-8454-11eb-9656-8a45c00ec89d.jpeg)
![image](https://user-images.githubusercontent.com/53358500/111057275-41251280-8454-11eb-9c14-1c3b7d690c73.jpeg)
![image](https://user-images.githubusercontent.com/53358500/111057279-44200300-8454-11eb-8728-f4392f61c528.jpeg)
![image](https://user-images.githubusercontent.com/53358500/111057284-46825d00-8454-11eb-98ca-01ec9324dae5.jpeg)

**Comment by animate1978** [comment]
Sorry for the delay.

II will look into this, I am not the author of the hair engines, that is all @Noizirom works, I personally haven't gotten issues with the code but then again I haven't tested fully either

**Comment by thelabcat** [comment]
Thanks. Sorry for the multiple posts.

Oh yes, testing. The one thing aperture science is good at. They should have put all their work into programming instead.￼

**Comment by animate1978** [comment]
It's ok for the multiple postings.

However I am not experiencing the problems you are describing so this is very odd...

**Comment by thelabcat** [comment]
I am using 1.7.8.42 and do not know how to check the current released version on GitHub.￼

**Comment by thelabcat** [comment]
Aha, I see in the commits page that the version is 1.7.8.50 so maybe the bug is fixed now.￼￼

**Comment by thelabcat** [comment]
Updated. Problem persists. ![image](https://user-images.githubusercontent.com/53358500/112365213-c5c22d00-8cad-11eb-92a4-7fee59e11e9b.jpeg)

Set for Cycles. Female elf.

**Comment by animate1978** [comment]
I am now getting this issue, I tried the Female Elf as you have above and sure enough the tooltips are garbage.

I am looking into this though seeing what could cause such problems.

**Comment by animate1978** [comment]
https://github.com/animate1978/MB-Lab/blob/7b3e5b5179b3398c431cf8484ba89e92743ab38b/__init__.py#L592

Seems that this chunk of code here is where the problems start? I could be wrong @Noizirom
